Makefile

```diff
@@ -50,8 +52,7 @@ $(STABLE_TARGETS) $(STAGING_TARGETS): $(TMPDIR)/.helm/repository/local/index.yam
 %/index.yaml: $(STABLE_TARGETS) $(STAGING_TARGETS)
 %/index.yaml: $(TMPDIR)/.helm/repository/local/index.yaml
 	@mkdir -p $(patsubst %/index.yaml,%,$@)
-	$(HELM) --home $(TMPDIR)/.helm repo index $(patsubst %/index.yaml,%,$@)
```

Comment:
Could we get `mesosphere` from `git remote origin`? I'd like to use this on my fork. Something like:

```
--url=https://$(shell git remote get-url origin | cut -d':' -f2 | cut -d'/' -f1).github.io/charts/$(patsubst...)
```

Reply:
higher up in  this file, could we have:
```
-	@git remote add publish git@github.com:mesosphere/charts >/dev/null 2>&1 || true
+	@git remote add publish $(shell git remote get-url origin) >/dev/null 2>&1 || true
```

Reply:
I'll fix these in my next PR, no worries.